src/components/Grid/GridComponent.vue

```diff
@@ -27,29 +27,15 @@
           <facet-option class="selectCol"><font-awesome-icon icon="arrow-down" /></facet-option>
         </td>
       </tr>
-
-      <tr
-        v-for="(row, rowIndex) in grid"
-        :key="rowIndex"
+      <grid-row
+        v-for="(variable, rowIndex) in variables"
+        :key="variable.id"
+        :label="variableName(variable)"
+        :selected="gridSelections[rowIndex]"
+        :counts="grid[rowIndex]"
+        @facetToggled="(colIndex) => toggle(rowIndex, colIndex)"
```

Comment:
could the grid-cell not send the var-id and assessment-id ( in addition to the row and col ) ?